### DISCUSSION TITLE
how to set Encoding or why I see only ???????

### DISCUSSION THREAD

**Discussion by VeronikaYurievnaA** [dompdf/dompdf#3068] [General]
All my files is UTF-8, but when PDF is ready, so I see only ?????? and no words. How can I to correct IT?

```
$options = new Options();
$options->set('defaultFont', 'Courier');
$dompdf=new Dompdf($options);
$dompdf->loadHtml('<h1>Красивый Тестик</h1>');
$dompdf->setPaper('A4', 'potrait');
$dompdf->render();
$dompdf->stream();

```

**Comment by bsweeney** [comment]
The core fonts (Helvetica, Times, Courier) and generic fonts (serif, sans-serif, monospace) currently only support Windows ANSI encoding. To get full character support you can use one of the bundled DejaVu fonts or another font with wider character support.

Relevant from the wiki:
* [About Fonts and Character Encoding](https://github.com/dompdf/dompdf/wiki/About-Fonts-and-Character-Encoding)
* [Unicode How To](https://github.com/dompdf/dompdf/wiki/UnicodeHowTo)
